config/environments/development.rb

```diff
@@ -83,5 +83,8 @@
   config.x.hses.auth_base = "https://staging.hses.ohs.acf.hhs.gov"
   config.x.hses.client_id = Rails.application.credentials.hses_client_id
   config.x.hses.client_secret = Rails.application.credentials.hses_client_secret
-  config.x.bypass_auth = ENV["BYPASS_AUTH"] == "true"
+  config.x.bypass_auth = ENV.fetch("CT_BYPASS_AUTH", "false") == "true"
```

Comment:
This might be overcomplicated now, since we're guaranteeing that `ENV["CT_BYPASS_AUTH"]` will have a value via `dotenv`

Reply:
[thought] i don't think we need it either way? won't it just return nil if it's not set, and `nil != "true"`

Reply:
> i don't think we need it either way?

that's true, it is not strictly needed for the reason you called out. I like having default values (every now and then having a default value instead of nil makes debugging things later easier) but this is now double default value'd